main.js

```diff
@@ -216,5 +229,44 @@ function renderDescription(surfboard) {
   $column2.appendChild($photoDiv)
   $photoDiv.appendChild($image)
 
+  $addCart.addEventListener('click', function (event) {
+    if (cart[surfboard.id] !== undefined) {
+      cart[surfboard.id].quantity += 1
+    }
+    else {
+      cart[surfboard.id] = {quantity: 1}
+    }
+    var $quantity = getCartQuantity(cart)
+    var $cart = renderCart($quantity)
+    $logo.appendChild($cart)
+  })
+
   return $panel
 }
+
+// brings you back to homepagen when click on logo
```

Comment:
Instead of a comment, extract a function called something like `returnToHome` and pass that to `addEventListener`.